doc/man3/OSSL_PROVIDER.pod

```diff
@@ -140,6 +144,10 @@ OSSL_PROVIDER_get_capabilities() return 1 on success, or 0 on error.
 OSSL_PROVIDER_load() and OSSL_PROVIDER_try_load() return a pointer to a
 provider object on success, or B<NULL> on error.
 
+OSSL_PROVIDER_do_all() returns 1 if all callback functions return 1, or 0 if
+any callback I<cb> returns 0; callback processing is stopped if a first
+callback returns 0.
```

Comment:
I'm not sure how to read "if a first callback". Do you mean "if any callback"?

Reply:
Well, that's the thing: The `do_all` code is calling the callback function in sequence for each provider found and stops doing so if one (the first) callback returns 0. So I considered the word "any" in the second part to be misleading as it somehow implies the callback would be called for all providers. But then again, I am no native English speaker :-]